### PR TITLE
Don't restrict the "Company Copyright" project setting to JUCE 5.2.0

### PR DESCRIPTION
**Pull request by McMartin**
@lethal-guitar, @MartyLake: please review, thanks!

### CODE REVIEW

File: cmake/Reprojucer.cmake [hunk 1]
```diff
@@ -1220,7 +1220,7 @@ function(jucer_project_end)
   endif()
 
   if(WIN32 AND NOT JUCER_PROJECT_TYPE STREQUAL "Static Library")
-    if(NOT(DEFINED JUCER_VERSION AND JUCER_VERSION VERSION_LESS 5.2))
+    if(DEFINED JUCER_COMPANY_COPYRIGHT)
```
Comment (lethal-guitar) [inline cmake/Reprojucer.cmake]: Who is defining `JUCER_COMPANY_COPYRIGHT`?

**Comment by lethal-guitar** [comment]
The new version of the code looks good, I don't know how it worked previously. Would jucer_copyright always be defined when using JUCE 5.2 or newer?

File: cmake/Reprojucer.cmake (hunk 1 above)
Comment (McMartin) [reply cmake/Reprojucer.cmake]: It's done by `jucer_project_settings()`  https://github.com/McMartin/FRUT/blob/1487c70dd613a05d1aa0cb3cbca130c5b08d8c42/cmake/Reprojucer.cmake#L199

when `tag` equals `COMPANY_COPYRIGHT`.

**Comment by McMartin** [comment]
@lethal-guitar I rebased the branch on the latest `master` (after merging #296) and I changed the code to be more accurate. Now `JUCER_COMPANY_COPYRIGHT` is used either when it's defined, or when using JUCE 5.2.0 and later.

**Pull request merged**